qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -126,6 +128,18 @@ def _build_single_proc_data_info(proc_data_id, data_type, samples):
     proc_info['pid'] = proc_data_id
     proc_info['data_type'] = data_type
     proc_info['processed_date'] = str(proc_info['processed_date'])
+    params = proc_data.processing_parameters.values
+    del params['input_data']
```

Comment:
Here I'm trying to replicate whatever was in the interface before. Before we were not showing which was the origin artifact, so that's why I'm just removing it here. Note that a lot of the changes here are kind of hackish because the system is way more powerful than the interface allows, so we need these kind of things until we change the interface.